content/chainguard/chainguard-images/verifying-chainguard-images-and-metadata-signatures-with-cosign.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Verifying Chainguard Images and Metadata Signatures with Cosign"
+linktitle: "Verifying with Cosign"
+aliases:
+  - /chainguard/chainguard-images/verifying-images-with-cosign
+type: "article"
+description: "A walkthrough of verifying Chainguard Images and metadata signatures with Cosign."
+date: 2024-03-18T08:59:52-07:00
+lastmod: 2024-03-18T08:59:52-07:00
+draft: false
+tags: ["Conceptual", "Chainguard Images", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 020
+toc: true
+---
+
+All Chainguard Images contain verifiable signatures and build-time SBOMs (software bill of materials), features that enable users to confirm the origin of each image built and have a detailed list of everything that is packed within.
+
+The following commands requires [cosign](https://docs.sigstore.dev/cosign/overview/) and [jq](https://stedolan.github.io/jq/) to be installed on your machine in order to download and verify image attestations. It will pull detailed information about all signatures found for the provided image.
```

Comment:
```suggestion
The following commands requires [Cosign](/open-source/sigstore/cosign/an-introduction-to-cosign/) and [jq](https://stedolan.github.io/jq/) to be installed on your machine in order to download and verify image attestations. It will pull detailed information about all signatures found for the provided image.
```